CODEOWNERS

```diff
@@ -51,4 +74,23 @@ platform/platform-paxweb-jettyconfig/*	@stustison
 
 platform/security/*		@stustison
 
+platform/security/security-migratable/*     @paouelle   @lambeaux
+
 platform/util/*			@stustison
+
+platform/migration/*            @paouelle   @lambeaux
+platform/platform-migratable/*  @paouelle   @lambeaux
+
+platform/platform-configuration/*       @lambeaux
+platform/platform-filter-clientinfo/*   @lambeaux
+
+
+###############################
+# ADMIN
```

Comment:
Please keep these modules within the above `PLATFORM` grouping, unless you wish to subsection all second tier modules. 

If you choose to subsection the second tier modules please refactor the existing blocks such that it is all consistent. And this subsection of platform should be titled `PLATFORM / ADMIN` in that case (and others likewise) if you choose to go that route.